src/SageFLBuilderServiceProvider.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Itineris\SageFLBuilder;
+
+use Illuminate\Support\Facades\View;
+use Itineris\SageFLBuilder\View\Composers\FLBuilder;
+use Roots\Acorn\ServiceProvider;
+
+use function Roots\base_path;
+
+class SageFLBuilderServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap any application services.
+     *
+     * @return void
+     */
+    public function boot(): void
```

Comment:
We aren't loading view from the sage-flbuilder Modules directory. This needs doing because our base GravityForm module can't load the frontend blade view.